x-pack/plugins/osquery/tsconfig.json

```diff
@@ -10,7 +10,7 @@
   "include": [
     // add all the folders contains files to be compiled
     "common/**/*",
-    "public/**/*",
+    "public/**/*.{json,tsx,ts}",
```

Comment:
can we instead exclude the files that are causing the issues?

Reply:
for some reason the glob wasn't picking up the schema json file until i explicitly added the `.json` extension. the other way i got this working was to add another entry `public/**/*.json`, which would be more permissive. WDYT?

Reply:
I think we can go with Canvas team approach like
https://github.com/lykkin/kibana/blob/osquery-autocomplete/x-pack/plugins/canvas/canvas_plugin_src/functions/server/demodata/get_demo_rows.ts#L9
and
https://github.com/lykkin/kibana/blob/osquery-autocomplete/x-pack/plugins/canvas/tsconfig.json#L27